proof-systems-shim/Cargo.toml

```diff
@@ -7,5 +7,11 @@ edition = "2021"
 license = "Apache-2.0"
 
 [dependencies]
-commitment_dlog = {git = "https://github.com/o1-labs/proof-systems", rev = "56517b55920ea7afc811baa541c859dadc8ae50a"}
-mina-hasher = {git = "https://github.com/o1-labs/proof-systems", rev = "56517b55920ea7afc811baa541c859dadc8ae50a"}
+commitment_dlog = {git = "https://github.com/o1-labs/proof-systems", rev = "6af257b793d80f900675acc01b16fd4bf6375268"}
+mina-curves = {git = "https://github.com/o1-labs/proof-systems", rev = "6af257b793d80f900675acc01b16fd4bf6375268"}
+plonk_circuits = {git = "https://github.com/o1-labs/proof-systems", rev = "6af257b793d80f900675acc01b16fd4bf6375268"}
+plonk_protocol_dlog = {git = "https://github.com/o1-labs/proof-systems", rev = "6af257b793d80f900675acc01b16fd4bf6375268"}
+
+mina-hasher = {git = "https://github.com/o1-labs/proof-systems", rev = "1f88a94eddba96b720c9214303fb2b4b0cd97597"}
```

Comment:
Possible to use the same rev for all crates?

Reply:
Not directly. The `plonk_*` ones no longer exist in the rev that makes the changes we cant. This definitely means we should no longer use them but we can make that change in another PR

Reply:
Actually, I might have removed the usage of stale proof-systems crates in https://github.com/ChainSafe/mina-rs/pull/198

Reply:
Yeah looks like it, that is merged now so all good.